express-notes-api/index.js

```diff
@@ -0,0 +1,76 @@
+
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const port = 3000;
+const data = require('./data.json');
+
+app.use(express.json());
+app.get('/api/notes', (req, res) => {
+  const arr = [];
+  for (const id in data.notes) {
+    arr.push(data.notes[id]);
+  }
+  res.status(200).send(arr);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  if (req.params.id < 0) {
+    res.status(400).send('Error: Number must be positive');
+  }
```

Comment:
Be sure to use `else if` or `return` to prevent your server from attempting to respond more than once to a request.